modules/src/erc1155_receiver.rs

```diff
@@ -9,41 +9,38 @@ pub struct Erc1155Receiver {}
 impl Erc1155Receiver {
     pub fn on_erc1155_received(
         &mut self,
-        #[allow(unused_variables)] operator: Address,
-        #[allow(unused_variables)] from: Address,
-        #[allow(unused_variables)] token_id: U256,
-        #[allow(unused_variables)] amount: U256,
-        #[allow(unused_variables)] data: Option<Bytes>
+        #[allow(unused_variables)] operator: &Address,
+        #[allow(unused_variables)] from: &Address,
+        #[allow(unused_variables)] token_id: &U256,
+        #[allow(unused_variables)] amount: &U256,
+        #[allow(unused_variables)] data: &Option<Bytes>
```

Comment:
to `Option<Bytes>`.

Reply:
Generally `&Option<T>` or `Option<&T>` should be used. Option is a zero-cost abstraction but is not a pointer, so its memsize is `size_of::<T>() + 1`